ibm/resource_ibm_cos_bucket.go

```diff
@@ -296,11 +343,10 @@ func resourceIBMCOSBucket() *schema.Resource {
 				},
 			},
 			"object_versioning": {
-				Type:          schema.TypeList,
-				Optional:      true,
-				MaxItems:      1,
-				ConflictsWith: []string{"retention_rule", "expire_rule"},
-				Description:   "Protect objects from accidental deletion or overwrites. Versioning allows you to keep multiple versions of an object protecting from unintentional data loss.",
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    1,
+				Description: "Protect objects from accidental deletion or overwrites. Versioning allows you to keep multiple versions of an object protecting from unintentional data loss.",
```

Comment:
why is conflict removed here we can proivide object_versioning along with retention_rule and expire_rule